Week1/homework/app.js

```diff
@@ -1,11 +1,303 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-undef */
+
 'use strict';
 
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
-  ];
+// 1.1
+// const bookTitles = [
+// Replace with your own book titles
+//   'war_and_peace',
+//   'the_great_gatsby',
+//   'great_expectations',
+//   'crime_and_punishment',
+//   'moby_dick',
+//   'hamlet',
+//   'madame_bovary',
+//   'jane_eyre',
+//   'pride_and_prejudice',
+//   'anna_karenina',
+// ];
+
+// 1.3
+
+// const ul = document.createElement('ul');
+
+// const createList = () => {
+//   for (let i = 0; i < bookTitles.length; i++) {
+//     const li = document.createElement('li');
+//     li.innerHTML = bookTitles[i];
+//     ul.appendChild(li);
+//   }
+// };
+// document.getElementById('renderList').appendChild(ul);
+
+// 1.4
+
+const myBooks = {
+  'War and Peace': {
```

Comment:
Usually keys are used as identifiers. I've noticed you're using them here for display as titles of the books. A more flexible approach is to keep the key as an identifier, e.g. `war_and_peace`, then add a key within the object like `title: 'War and Peace'`.